src/index.html

```diff
@@ -22,6 +22,409 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#apple"
+              class="header__nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#sumsung"
```

Comment:
The text 'sumsung' seems to be a typo, it should be 'Samsung'. Always ensure proper spelling for better user experience and professionalism.